setup.py

```diff
@@ -281,6 +281,7 @@ def finalize_options(self):
         'scipy >= 0.7.0',
         'six >= 1.5.0',
         'smart_open >= 1.2.1',
+        'morfessor==2.0.2alpha4',
```

Comment:
-1: why should this be a hard dependency??

Has this been thoroughly vetted, by @tmylk and @gojomo ? Is `morfessor` really a core, indispensable part of gensim now, with all that it entails (we'll have to support its installation problems etc).